mapswipe_workers/mapswipe_workers/mapswipe_workers.py

```diff
@@ -101,6 +104,7 @@ def run_firebase_to_postgres() -> list:
     update_data.update_user_data()
     update_data.update_project_data()
     project_ids = transfer_results.transfer_results()
+    [send_progress_notification(project_id) for project_id in project_ids]
```

Comment:
For me a simple for loop would be more readbale. Its unsual to do a list comprehension without saving the resulting list. At first it seems more like a potential bug.